### DISCUSSION TITLE
Need help

### DISCUSSION THREAD

**Discussion by md-zaikin** [RavenSystem/esp-homekit-devices#2119] [MEPLHAA Scripts]
Hello guys, I need help - I can’t convert the following configuration, yesterday I updated the device and realized that now I need a different code. Thank you all in advance.

{"c":{"l":13,"b":[{"g":0,"t":5}]},"a":[{"0":{"r":[{"g":12}]},"1":{"r":[{"g":12,"v":1}]},"b":[{"g":4,"p":0},{"g":4,"t":0,"p":0},{"g":0,"t":1}],"s":0},{"t":1,"b":[{"g":1,"t":0},{"g":1,"t":1}],"s":0}]}
 and 
{"c":{"l":13,"b":[{"g":0,"t":5}]},"a":[{"0":{"r":[{"g":12}]},"1":{"r":[{"g":12,"v":1}]},"b":[{"g":4,"p":0},{"g":4,"t":0,"p":0}],"s":0},{"t":1,"b":[{"g":1,"t":0},{"g":1,"t":1}],"s":0},{"t":1,"b":[{"g":3,"t":1},{"g":3,"t":0}],"s":0}]}

**Comment by RavenSystem** [comment]
```json
{"c":{"io":[[[12,13],2],[[0,4],6],[[1],6,1]],"l":13,"b":[[0,5]]},"a":[{"0":{"r":[[12]]},"1":{"r":[[12,1]]},"b":[[4,0],[4],[0]]},{"b":[[1,0],[1]]}]}
```

```json
{"c":{"io":[[[12,13],2],[[0,4],6],[[1,3],6,1]],"l":13,"b":[[0,5]]},"a":[{"0":{"r":[[12]]},"1":{"r":[[12,1]]},"b":[[4,0],[4]]},{"t":1,"b":[[1,0],[1]]},{"b":[[3,0],[3]]}]}
```
